docker/install/ubuntu_install_tensorflow_aarch64.sh

```diff
@@ -26,4 +26,4 @@ apt-install-and-clear -y --no-install-recommends libhdf5-dev
 pip3 install \
     numpy==1.23.5 \
     keras==2.9 \
-    tensorflow-aarch64==2.9.1
+    tensorflow-aarch64==2.9.3
```

Comment:
We could use `~=` to make sure patch updates are automatically used in the future